requirements.txt

```diff
@@ -1,10 +1,10 @@
-netcdf4
-dask
-numpy
-pandas
-pyyaml
-xarray
-xgboost
-scikit-learn<1.6
-scipy
+netcdf4==1.6.5
+dask==2024.5.0
+numpy==1.26.4
+pandas==2.2.2
+pyyaml==6.0.1
+xarray==2023.6.0
+xgboost==1.7.6
+scikit-learn==1.3.0
+scipy==1.13.1
```

Comment:
I'd suggest using open version requirements rather than exact pinning, because this may be a problem for future compatibility. NumPy for instance is already at version 2.0 and only since version 2.0 is Python 3.13 supported, which means that right now with these version specifiers you already cannot install Abil with the latest version of Python. If these are minimum supported versions, I'd suggest:
```
netcdf4~=1.6
dask>=2024.5
numpy~=2.0         ; python_version >= '3.10'
numpy~=1.26       ; python_version < '3.9'
pandas~=2.2
pyyaml~=6.0
xarray>=2023.6.0
xgboost~=1.7
scikit-learn~=1.3
scipy~=1.13
```
in accordance with [SemVer](https://semver.org).